src/python/tekst_abc/tekst_abc.md

```diff
@@ -130,19 +129,18 @@ Dette må du gjøre:
 
 - [ ] Lagre filmtittelen i en variabel.
 
-- [ ] Manipuler strengen slik at resultatet blir som beskrevet over.
+- [ ] Manipuler teksten slik at resultatet blir som beskrevet over.
 
-- [ ] Skriv ut den nye strengen.
+- [ ] Skriv ut den nye teksten.
 
 
 # Telling av tekst {.activity}
 
 
 # {.protip}
 
-Ved hjelp av `s.count()` kan vi finne ut om en streng inneholder en bestemt
-tekst og hvor mange ganger den finnes i strengen. For eksempel så inneholder
-strengen `"Hei verden!"` teksten `"verden"` en gang.
+Ved hjelp av `s.count()` kan vi finne ut om en tekst inneholder en bestemt
+tekst og hvor mange ganger den finnes i teksten. For eksempel så inneholder `"Hei verden!"` teksten `"verden"` en gang.
```

Comment:
Kan vere lurt å finne noko anna for det som opphavleg var "tekst" i dette avsnittet. Til dømes "... kan vi finne ut om en tekst inneholder et bestemt ord og hvor mange ganger det finnes i teksten. For eksempel inneholder `"Hei verden!"` ordet `"verden"` én gang."